install_config/master_node_configuration.adoc

```diff
@@ -1762,38 +1762,58 @@ $ openshift start node --config=/openshift.local.config/node-<node_hostname>/nod
 |===
 
 You can control which INFO messages are logged by setting the loglevel option in the in xref:../admin_guide/manage_nodes.adoc#modifying-nodes[node configuration files], 
-the *_/etc/origin/master/master-config.yaml_* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
+the *_/etc/sysconfig/atomic-openshift-node_* file. Configuring the logs to collect all messages can lead to large logs that are difficult to interpret and can take up excessive space. Collecting all messages should only be used in debug situations.
 
 [NOTE]
 ====
 Messages with FATAL, ERROR, WARNING and some INFO severities appear in the logs regardless of the log configuration.
 ====
 
-You can view logs for the master or the node system using the following command:
+*View node logs*
+
+To view logs for the node system, use the following command:
 
 ----
 # journalctl -r -u <journal_name>
 ----
 
+*View master logs*
+
+To view logs for the master, use the following command:
+
+----
+# /usr/local/bin/master-logs <component> <container name> <1>
+----
+
```

Comment:
> <1>

You intend to place then ending `<1>` after `journalctl -r -u <journal_name>`? If yes, move below `Use the `-r` option to show the newest entries first.` up, too

Reply:
@xingxingxia Thank you for catching this. The <1> is for the `<component> <container name>` variables. I moved the `-r` sentence up to the command for viewing the log nodes. The `-r` doesn't seem to work for the `/usr/local/bin/master-logs <component> <container name>` command.